bootstrap.lisp

```diff
@@ -0,0 +1,25 @@
+;;;; Creates executable for linux and windows.
+;;; usage: sbcl --load bootstrap.lisp
+;;; emacs: M-! sbcl --load bootstrap.lisp
+
+(ql:quickload :aark)
+
+#+(and (not windows) (or win32 mswindows))(pushnew :windows *features*)
```

Comment:
Cool!
But I think you really should move `(pushnew  :windows *features*)` to the newline.